src/backend.rs

```diff
@@ -399,9 +399,9 @@ impl State {
                 // Ensure that we release the borrow before calling `cb`.
                 let borrow = &*service.borrow();
                 let view = ServiceView::new(borrow);
-                matches = selectors.iter().find(|selector| {
+                matches = selectors.iter().any(|selector| {
```

Comment:
Definitely works for me, but why do we need this change? (and all the others of the same kind)